Projects/1-Beginner/Password-Strength-Checker.md

```
@@ -0,0 +1,39 @@
+# Password Strength Checker Idea
+
+**Tier:** 1-Beginner
+
+A web application that helps users create strong passwords by providing real-time feedback on password strength.
+
+## User Stories
+
+-   [ ] User can enter a password into an input field
+-   [ ] User can see strength indicators as they type:
+    - Password length (minimum 8 characters)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)